judo-tatami-psm2asm/src/main/epsilon/transformations/asm/psmToAsm.etl

```diff
@@ -19,5 +19,5 @@ post {
         }
     }
 
-    asmUtils.validateUniqueXmiids();
+    // asmUtils.validateUniqueXmiids();
```

Comment:
Will this commented-out line be used later? If not, we should remove it.

Reply:
No, it can be used when developing transformations.